go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	github.com/lightningnetwork/lightning-onion v1.2.1-0.20240712235311-98bd56499dfb
```

Comment:
Those files were changed when I did `go get -u github.com/lightningnetwork/lnd/fn` to update the Fn to use isEmpty function once https://github.com/lightningnetwork/lnd/pull/9159 is merged